test/conformance/main_test.go

```diff
@@ -51,7 +51,7 @@ func TestMain(m *testing.M) {
 	os.Exit(func() int {
 		test.InitializeFlags()
 		eventingtest.InitializeEventingFlags()
-		projectID = os.Getenv(lib.ProwProjectKey)
+		projectID, e2e_test := os.LookupEnv(lib.ProwProjectKey)
```

Comment:
This is too clever IMO...

also not supposed to have underscores in Go variable names

would prefer

```
projectID, projectProvided :=
```

or 

```
// If there's no project ID environment variable, it's probably a build test
projectID, e2eTest :=
```

Reply:
Done.